example/index.js

```diff
@@ -0,0 +1,23 @@
+// Load configured OpenTelemetry SDK first.
+require('./opentelemetry')
+
+const fastify = require('fastify')
+const fastifyOpentelemetry = require('../index.js')
```

Comment:
does this need the file / ext? 
```suggestion
const fastifyOpentelemetry = require('../')
```

Reply:
Oh haha, that was VSCode updating the path when I moved the file around. Good catch 👍